compiler/modules/types/core.bal

```diff
@@ -790,6 +797,10 @@ public function isNever(SemType t) returns boolean {
     return t is BasicTypeBitSet && t == 0;
 }
 
+public function isUndef(SemType t) returns boolean {
```

Comment:
Now that SemType is anydata, do we need this (and `isNever`)?

Reply:
I think I didn't get this point. `isNever()` is used in a lot of places ATM to see if a sem type is `never`. However, isUndef is only used in `isUndefInner()`. Since we can't get rid of `isNever()`. Isn't that okay to have a `isUndef()` too?

Reply:
Removed with [8b9d93d](https://github.com/ballerina-platform/nballerina/pull/1140/commits/8b9d93d6c90ad619fb64ec8a12a1c95b2aac8ddc)